test/framework/e2e_framework.go

```diff
@@ -68,6 +68,10 @@ func NewE2eFramework(t *testing.T, kubeconfigFile string, useDse bool, controlPl
 		cqlshBinLocation = "/opt/dse/bin/cqlsh"
 	}
 
+	if useHcd {
```

Comment:
Issue: I'd rather see this value string interpolated from a serverType field given the expanding number of server types we are supporting.